src/components/learner-credit-management/requests-tab/AmountCell.jsx

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 import { formatPrice } from '../data';
 
 const AmountCell = ({ row }) => {
-  const formattedContentPrice = formatPrice(row.original.amount);
+  const formattedContentPrice = formatPrice(row.original.amount / 100);
```

Comment:
shouldn't this also be replaced with `course_price?`

Reply:
Actually in the DataTable the cell name is Amount so i keep it amount here instead of course_price to avoid any confusions.